client/src/components/SettingsPage/SettingsMenu.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <ul>
+    <li><router-link to='/settings/editUserInfo/'>User info</router-link></li>
```

Comment:
Please beware that sometimes you might need to create this array (or object with subobjects) in computed instead of data - this is not the case but it might be useful in case of dynamic items that really depend on let's say some getter in store.